combat-trainer.lic

```diff
@@ -1999,6 +2036,62 @@ class GameState
     @attack_overrides[weapon_skill]
   end
 
+  def set_currently_whirlwinding(setting)
```

Comment:
Since you defined `currently_whirlwinding` in the attr_accessor list, you don't need this set function. You can `game_state.currently_whirlwinding = true` in a process.